app/_data/docs_nav_ee_1.3-x.yml

```diff
@@ -139,6 +141,8 @@
         url: /admin-api/vitals/vitals-prometheus-strategy
   - text: Audit Logging
     url: /admin-api/audit-log
+  - text: Keyring and Data Encryption
```

Comment:
Looks like we are linking this in the nav twice? was this on purpose?

Reply:
@p0pr0ck5 , these seem to point to different paths.

Reply:
I see now. One is an overview, the other is API reference.